packages/esbuild-plugin/src/index.ts

```diff
@@ -43,15 +45,20 @@ export function vanillaExtractPlugin({
       build.onLoad(
         { filter: /.*/, namespace: vanillaCssNamespace },
         async ({ path }) => {
-          let { source } = await getSourceFromVirtualCssFile(path);
+          let { source, fileName } = await getSourceFromVirtualCssFile(path);
 
           if (typeof processCss === 'function') {
             source = await processCss(source);
           }
 
+          const rootDir = build.initialOptions.absWorkingDir ?? process.cwd();
+
+          const resolveDir = dirname(join(rootDir, fileName));
```

Comment:
may absWorkingDir end with '/'? probably
```
dirname(resolve(rootDir, fileName));
```
is the best option?

Reply:
`path.join` handles the removal of duplicate slashes for you. I think `join` is better suited here.